File: pset6/mario/mario.py
from cs50 import get_int

# A function to print mario piramid
# It takes height as a parameter and print the piramid


def print_mario_piramid(height):
    for x in range(1, height+1):
        print(" " * (height - x) + ("#" * x) + "  " + ("#" * x) + " " * (height - x))


# initializing height with any negative number or zero.
h = -1

while h <= 0 or h > 8:
    # Prompting user to enter piramid's height
    h = get_int("Height: ")

# printing the piramid if user enters a desired valued of height
# The piramid is printed by the print_mario_piramid() function

print_mario_piramid(h)

File: pset1/mario/mario.c
#include<cs50.h>
#include<stdio.h>

int main(void)
{
    // Prompting a user to enter height of the mario piramid
    int h =  get_int("Height: ");

    // checking if the value entered ranges between 1 and 8
    if (h < 1 || h > 8)
    {
        while (true)
        {
            h = get_int("Height: ");
            if (h >= 1 && h <= 8)
            {
                break;
            }
        }
    }
    // Printing the mario piramid
    for (int i = 1; i <= h ; i++)
    {
        for (int j = 1; j <= h - i; j++)
        {
            printf(" ");
        }

        for (int m = 1; m <= i; m++)
        {
            printf("#");
        }
        printf("\n");
    }
    // end

}